quotas_sample.txt

```diff
@@ -0,0 +1,6 @@
+developers /filestore/departments/developers /mnt/departments/developers 30.0
```

Comment:
It’s not specific. That’s a bogus example. 


On Jun 16, 2016, at 2:26 PM, Michael Murray <notifications@github.com> wrote:


In 
quotas_sample.txt:
> @@ -0,0 +1,6 @@
> +developers /filestore/departments/developers /mnt/departments/developers 30.0

@carlilek this is specific to your installation, can you remove this from the PR please?
—
You are receiving this because you were mentioned.
Reply to this email directly, 
view it on GitHub, or 
mute the thread.